config/rbac/role.yaml

```diff
@@ -13,6 +13,12 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
```

Comment:
Can this also be restricted by resource names?

Reply:
It can! Thanks